pyfolio/timeseries.py

```diff
@@ -547,24 +547,21 @@ def rolling_beta(returns, factor_returns,
         return out
 
 
-def rolling_regression(returns, factor_returns=None,
+def rolling_regression(returns, factor_returns,
                        rolling_window=APPROX_BDAYS_PER_MONTH * 6,
                        nan_threshold=0.1):
     """
-    Computes rolling Fama-French single factor betas using a multivariate
-    linear regression (separate linear regressions is problematic because
-    the Fama-French factors are confounded).
-
-    Specifically, returns rolling betas to SMB, HML, and UMD.
+    Computes rolling single factor betas using a multivariate linear regression
```

Comment:
I'm a bit confused by the expression "single factor betas using a multivariate linear regression". @twiecki is that accurate terminology? Naively I would have expected that "single factor beta" implies a univariate regression.